### PR TITLE
Update antiadblock.txt

### PR DESCRIPTION
**Pull request by ghost**
https://github.com/AdguardTeam/AdguardFilters/issues/87713
https://github.com/AdguardTeam/AdguardFilters/issues/87714
https://github.com/AdguardTeam/AdguardFilters/issues/87715
https://github.com/AdguardTeam/AdguardFilters/issues/87716
https://github.com/AdguardTeam/AdguardFilters/issues/87717
https://github.com/AdguardTeam/AdguardFilters/issues/87719
https://github.com/AdguardTeam/AdguardFilters/issues/87720
https://github.com/AdguardTeam/AdguardFilters/issues/87721
https://github.com/AdguardTeam/AdguardFilters/issues/87805
https://github.com/AdguardTeam/AdguardFilters/issues/87806
https://github.com/AdguardTeam/AdguardFilters/issues/87807
https://github.com/AdguardTeam/AdguardFilters/issues/87808
https://github.com/AdguardTeam/AdguardFilters/issues/87809
https://github.com/AdguardTeam/AdguardFilters/issues/87810
https://github.com/AdguardTeam/AdguardFilters/issues/87811
https://github.com/AdguardTeam/AdguardFilters/issues/87812
https://github.com/AdguardTeam/AdguardFilters/issues/87813
https://github.com/AdguardTeam/AdguardFilters/issues/87814
https://github.com/AdguardTeam/AdguardFilters/issues/87815
https://github.com/AdguardTeam/AdguardFilters/issues/87816
https://github.com/AdguardTeam/AdguardFilters/issues/87817
https://github.com/AdguardTeam/AdguardFilters/issues/87818
https://github.com/AdguardTeam/AdguardFilters/issues/87819
https://github.com/AdguardTeam/AdguardFilters/issues/87820
https://github.com/AdguardTeam/AdguardFilters/issues/87821
https://github.com/AdguardTeam/AdguardFilters/issues/87822
https://github.com/AdguardTeam/AdguardFilters/issues/87823
https://github.com/AdguardTeam/AdguardFilters/issues/87824
https://github.com/AdguardTeam/AdguardFilters/issues/87825
https://github.com/AdguardTeam/AdguardFilters/issues/87826

### CODE REVIEW

File: AnnoyancesFilter/sections/antiadblock.txt [hunk 1]
```diff
@@ -536,6 +536,7 @@ autoblog.com#@#.ad
 !
 !
 !
+agrihaber04.com,afyonhaberturk.com,nehaberankara.com,mansetburdur.com,bolgesellig.com,telgrafgazetesi.com,burokratika.com,balikesirartihaber.com,ajansbalikligol.com,karar67.com,gazetemalatya.com,vansesigazetesi.com,kocaelisabah.com,antalyadanhaberler.com,aydinyeniufuk.com.tr,yerelvanhaber.com,a2teker.com,kibrishakikat.com,izmirtime35.com,sancakplus.com,marasfisilti.com,burdurgazetesi.com,zeytinburnuhaber.org,seffafbelediyecilik.com,denizli20haber.com,akdenizdeyeniyuzyil.net,ulakci.com,egegundem.com.tr,isdunyasindakadin.com,haberimizvar.net#%#//scriptlet('abort-on-property-read', 'isAdBlockActive')
```
Comment (zloyden) [inline AnnoyancesFilter/sections/antiadblock.txt:539]: This rule is ok, but we also have platforms that don't support advanced rules. They are called content blockers.
Could you check these sites with this rule `@@/advert.js$domain=`? After `$domain=` you should add the list of sites.
I have checked only two of them and it seems ok.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/antiadblock.txt (hunk 1 above)
Comment (IThinkIAmCool) [reply AnnoyancesFilter/sections/antiadblock.txt:539]: Do you want it replaced with `@@/advert.js$domain=agrihaber04.com|afyonhaberturk.com|nehaberankara.com|mansetburdur.com|bolgesellig.com|telgrafgazetesi.com|burokratika.com|balikesirartihaber.com|ajansbalikligol.com|karar67.com|gazetemalatya.com|vansesigazetesi.com|kocaelisabah.com|antalyadanhaberler.com|aydinyeniufuk.com.tr|yerelvanhaber.com|a2teker.com|kibrishakikat.com|izmirtime35.com|sancakplus.com|marasfisilti.com|burdurgazetesi.com|zeytinburnuhaber.org|seffafbelediyecilik.com|denizli20haber.com|akdenizdeyeniyuzyil.net|ulakci.com|egegundem.com.tr|isdunyasindakadin.com|haberimizvar.net`?

Comment (zloyden) [reply AnnoyancesFilter/sections/antiadblock.txt:539]: Not replace - also add for content blockers.
1. Don't remove the scriptlet rule.
2. Check an exception rule on these sites.
3. If it's ok, you can add it to the Annoyances - antiadblock.txt - look at the end of the file - you will see the section. Add the exception rule here.
```
! Common safari_cb_affinity(general)
!#safari_cb_affinity(general)
```

Comment (IThinkIAmCool) [reply AnnoyancesFilter/sections/antiadblock.txt:539]: With the other rule, on `https://www.aydinyeniufuk.com.tr` there is something on the right side.

**Review by zloyden — APPROVED** [review]

File: AnnoyancesFilter/sections/antiadblock.txt (hunk 1 above)
Comment (zloyden) [reply AnnoyancesFilter/sections/antiadblock.txt:539]: Ok, fixed.

**Pull request merged**